### PR TITLE
Even more cleanup

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -7,7 +7,6 @@ provisioner:
 platforms:
   - name: centos-6.8
   - name: centos-7.2
-  - name: debian-7.11
```
Comment (shortdudey123) [inline .kitchen.yml]: Fails on 7.11? or removing it to match dokken testing? if so, whys that?

Comment (tas50) [inline .kitchen.yml]: Lacks packages

Comment (shortdudey123) [inline .kitchen.yml]: ah

Comment (tas50) [inline .kitchen.yml]: 6 and 8 have packages, but not 7. It's really weird

**Pull request merged**